JenkinsfileRT

```diff
@@ -25,26 +25,26 @@ pytest_basetemp = "test_outputs"
 // Configure artifactory ingest
 data_config = new DataConfig()
 data_config.server_id = 'bytesalad'
-data_config.root = '${PYTEST_BASETEMP}'
+data_config.root = '${pytest_basetemp}'
 data_config.match_prefix = '(.*)_result' // .json is appended automatically
 
 // Build and test with python 3.6 and released dependencies from astroconda
 bc = new BuildConfig()
 bc.nodetype = 'jwst'
-bc.name = 'stable-deps'
+bc.name = 'stable'
 bc.env_vars = env_vars
-bc.conda_ver = '4.6.14'
+bc.conda_ver = '4.7.5'
 bc.conda_packages = [
     "python=${python_version}",
 ]
 bc.build_cmds = [
+    "pip install numpy",
```

Comment:
I thought line 42 was supposed to install absolutely everything that's needed for the jwst package. I guess there's something unique about numpy and the order in which it gets installed?

Reply:
It does install numpy, but it bootstraps it.  Because currently `spherical_geometry`, and a two of the `stsci.foo` packages need `numpy` but don't currently specify that in a way that works, they fail on the initial installation, but everything else gets installed, including `numpy`, then `pip` bootstraps back to those 3 failed installations and installs them again, successfully.  When new releases of these packages specifying `numpy` in a `pyproject.toml` file appear, this shouldn't be necessary.  This basically just speeds up the installation, maybe.